MapboxNavigation/RouteVoiceController.swift

```diff
@@ -282,7 +286,8 @@ public protocol VoiceControllerDelegate {
      
      - parameter voiceController: The voice controller that experienced the interruption.
      - parameter instruction: The spoken instruction that will be said.
+     - parameter routeProgress: The current `RouteProgress` when the instruction was generated.
```

Comment:
The instruction was generated on the server at request time. This is the route progress just before the instruction is scheduled to be spoken.